# Generated by Django 2.1 on 2019-11-08 12:17

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('leads', '0038_auto_20190708_1212'),
    ]

    operations = [
        migrations.AddField(
            model_name='historicallead',
            name='before_reactivated_finish_date',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='historicallead',
            name='before_reactivated_result',
            field=models.CharField(blank=True, choices=[('not_available', 'No disponible'), ('negative', 'Negative'), ('reserved_vehicle', 'Vehicle reserved'), ('unreachable', 'Unreachable'), ('wrong', 'Duplicado')], max_length=255, null=True),
        ),
        migrations.AddField(
            model_name='historicallead',
            name='is_reactivated',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='historicallead',
            name='reactivated_date',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='lead',
            name='before_reactivated_finish_date',
            field=models.DateTimeField(blank=True, null=True),
        ),
        migrations.AddField(
            model_name='lead',
            name='before_reactivated_result',
            field=models.CharField(blank=True, choices=[('not_available', 'No disponible'), ('negative', 'Negative'), ('reserved_vehicle', 'Vehicle reserved'), ('unreachable', 'Unreachable'), ('wrong', 'Duplicado')], max_length=255, null=True),
        ),
        migrations.AddField(
            model_name='lead',
            name='is_reactivated',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='lead',
            name='reactivated_date',
            field=models.DateTimeField(blank=True, null=True),
        ),
    ]
